glotaran/utils/test/test_io.py

```diff
@@ -157,3 +158,13 @@ def test_relative_posix_path(tmp_path: Path, rel_file_path: str):
     )
 
     assert rel_result_no_coomon == f"../{rel_file_path}"
+
+
+@pytest.mark.skipif(not sys.platform.startswith("win32"), reason="Only needed for wondows")
+def test_relative_posix_path_windows_diff_drives():
+    """os.path.relpath doesn't cause crash when files are of different drives."""
```

Comment:
when files are *on* different drives

Reply:
Fixed in 4d6301c0a66e31de369eba0c044c162f2f81c60e